test/alternator/test_metrics.py

```diff
@@ -130,6 +131,31 @@ def test_batch_write_item(test_table_s, metrics):
         test_table_s.meta.client.batch_write_item(RequestItems = {
             test_table_s.name: [{'PutRequest': {'Item': {'p': random_string(), 'a': 'hi'}}}]})
 
+def test_batch_write_item_histogram(test_table_s, metrics):
+    with check_increases_metric_exact(metrics, "scylla_alternator_batch_item_count_histogram_bucket", [[0, {'op': 'BatchWriteItem', 'le': '2.000000'}], [1, {'op': 'BatchWriteItem', 'le': '3.000000'}]]):
+        test_table_s.meta.client.batch_write_item(RequestItems = {
+            test_table_s.name: [{'PutRequest': {'Item': {'p': random_string(), 'a': 'hi'}}}, {'PutRequest': {'Item': {'p': random_string(), 'a': 'hi1'}}}, {'PutRequest': {'Item': {'p': random_string(), 'a': 'hi1'}}}]})
+    items = [{'PutRequest': {'Item': {'p': random_string(), 'a': 'hi'}}} for i in range(100)]
+    items2 = [{'PutRequest': {'Item': {'p': random_string(), 'a': 'hi'}}} for i in range(100)]
+    with check_increases_metric_exact(metrics, "scylla_alternator_batch_item_count_histogram_bucket", [[0, {'op': 'BatchWriteItem', 'le': '86.000000'}], [2, {'op': 'BatchWriteItem', 'le': '103.000000'}]]):
```

Comment:
I'm not crazy on the fact that this test enshrines these silly numbers 86 and 103, which might as well change in a later release.
I think a more "stable" test would have been to write a function that checks all the buckets and expect only the first one higher than 100 to be increased.

Reply:
You are right on the one hand, but we end up with a test that does not test what we really have, it's true that if someone changes the histogram implementation, the test would break, but isn't that a good thing?

Reply:
No, it's not a good thing because it's annoying when you try to improve some code (e.g., histograms) and it turns out that people built mountains of tests that enshrine all sorts of implementation details that were never promised or documented.
If we agree it's not a good thing, I can agree that you don't need to spend another hour writing Python code that will make this test work regardless of whether the histograms have buckets at 86 and 103 - or 89 and 107.